src/sliceDataStorage.h

```diff
@@ -156,6 +156,7 @@ class SliceMeshStorage : public SettingsMessenger // passes on settings from a M
     GCodePathConfig insetX_config;
     GCodePathConfig skin_config;
     std::vector<GCodePathConfig> infill_config;
+    std::vector<int> infill_angles;
```

Comment:
Please add documentation which says that the values are used on censecutive layers:
`//!< a list of angle values (in degrees) which is cycled through to determine the infill angle of each layer`